apps/frontend/core-dapp/src/features/trade/open-trade-utils.ts

```diff
@@ -0,0 +1,215 @@
+import { BigNumber } from 'ethers'
+import { DepositTradeHelperStore } from '../../stores/DepositTradeHelperStore'
+import { CollateralStore } from '../../stores/CollateralStore'
+
+type Props = {
+  collateralStore: Pick<
+    CollateralStore,
+    'balanceOfSignerInWstEth' | 'depositFee' | 'percentDenominator'
+  >
+  depositTradeHelperStore: Pick<
+    DepositTradeHelperStore,
+    'depositAndTradeFee' | 'percentDenominator' | 'tradeFee'
+  >
```

Comment:
so after watching your video explanation, I think we should just do this:

```suggestion
type Props = {
  collateralStore: CollateralStore
  depositTradeHelperStore: DepositTradeHelperStore
```